pytest-embedded-wokwi/pytest_embedded_wokwi/dut.py

```diff
@@ -0,0 +1,25 @@
+from typing import AnyStr
+
+from pytest_embedded.dut import Dut
+
+from .wokwi_cli import WokwiCLI
+
+
+class WokwiDut(Dut):
+    """
+    Wokwi DUT class
+    """
+
+    def __init__(
+        self,
+        wokwi: WokwiCLI,
+        **kwargs,
+    ) -> None:
+        self.wokwi = wokwi
+
+        super().__init__(**kwargs)
+
+        self._hard_reset_func = self.wokwi._hard_reset
+
+    def write(self, s: AnyStr) -> None:
+        self.wokwi.write(s)
```

Comment:
Is there a way to do anything else interesting with the `dut`, other than serial I/O?

For example if there is an LED in the diagram, can we get its state using something like `dut.get_board_state('led', 'level')`? Or, get the image from the emulated LCD? Or modify the state of a switch or another control on the board from the test case?

Not saying this is a must have to merge the PR, but I think it is features like this that would be unique for Wokwi and not available with the alternatives (real HW or QEMU).

Reply:
I'll look into this once we've figured out the other parts.

Reply:
created #233 to track it